test/scripts/hexo/locals.js

```diff
@@ -24,10 +24,10 @@ describe('Locals', () => {
     locals.set('foo', () => 'foo');
 
     // cache should be clear after new data is set
-    should.not.exist(locals.cache.foo);
+    should.not.exist(locals.cache.cache.foo);
```

Comment:
Currently `Cache()` has no `dump()` function, but the cached content could be find through `.cache` property.

Will be refactor `cache.has()` later.